src/nativeAssetManager.js

```diff
@@ -160,14 +161,26 @@ export function newNativeAssetManager(win) {
   /*
    * Entry point to search for placeholderes and set up postmessage roundtrip
    * to retrieve native assets. Looks for placeholders for the given adId and
-   * fires a callback after the native html is updated.
+   * fires a callback after the native html is updated. If no placeholders found
+   * and requestAllAssets flag is set in the tag, postmessage roundtrip
+   * to retrieve native assets that have a value on the corresponding bid
    */
   function loadAssets(adId, cb) {
-    const placeholders = scanForPlaceholders(adId);
+    const placeholders = scanForPlaceholders(adId), flag = (typeof win.pbNativeData !== 'undefined');
+
+    if (flag && win.pbNativeData.hasOwnProperty('assetsToReplace')) {
+        win.pbNativeData.assetsToReplace.forEach((asset) => {
+          const key = (asset.match(/hb_native_/i)) ? asset : NATIVE_KEYS[asset];
+          if (key) {placeholders.push(key);}
+        });
+    }
 
     if (placeholders.length > 0) {
       callback = cb;
       requestAssets(adId, placeholders);
+    }else if (flag && win.pbNativeData.hasOwnProperty('requestAllAssets') && win.pbNativeData.requestAllAssets) {
```

Comment:
space after closing brackets

Reply:
Fixed these. Not sure how they didnt update last time around